chart/templates/secret.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
 type: Opaque
 data:
   {{- $existingSecret := lookup "v1" "Secret" .Release.Namespace "bootstrap-secret" }}
-  {{- if $existingSecret }}
-  bootstrapPassword: {{ $existingSecret.data.bootstrapPassword }}
-  {{- else }}
-  bootstrapPassword: {{ .Values.bootstrapPassword | b64enc | quote }}
-  {{- end }}
\ No newline at end of file
+    {{- if and $existingSecret $existingSecret.data }}
```

Comment:
Would recommend to put this into a separate PR. If anything go wrong reverting might be easier.

Reply:
I agree, separate pr as it's not part of this fix or issue.

Reply:
https://github.com/rancher/rancher/pull/34249